kubeadm/buildx.psm1

```diff
@@ -0,0 +1,60 @@
+function Set-Builder()
+{
+    $env:DOCKER_CLI_EXPERIMENTAL = "enabled"
+    & docker buildx create --name img-builder --use --driver docker-container --driver-opt image=moby/buildkit:v0.7.2
+}
+
+function New-Build()
+{
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory = $true)]
+        [string]$name,
+        [Parameter(Mandatory = $true)]
+        [ValidateSet("docker", "registry")]
+        [string]$output,
+        [Parameter(Mandatory = $true)]
+        [string[]]$args
+    )
+
+    $command = "docker buildx build --platform windows/amd64 --output=type=$output -f Dockerfile -t $name"
+    foreach($arg in $args)
+    {
+        $command = "$command --build-arg=$arg"
+    }
+    $command = "$command ."
+    Write-Host $command
+    Invoke-Expression $command
+}
+
+function Push-Manifest([string]$name, [string[]]$items, [string[]]$bases)
+{
+    $command = "docker manifest create $name";
```

Comment:
during my local testing if this is called multiple times then it will continually append the manifest lists. This is not an issue in CI as it is not persisted but if someone wanted to use this to push to custom repo they could run into it.  

If an existing manifest is present on the machine of the same name can we clear it?  This should either print a warning or maybe even take a flag?

Reply:
I didn't think about it. Ok. I will try to remove previous manifest if it exists and warning about it.

Reply:
Check exists manifest has been done.